src/test/java/com/google/step/youtube/DescriptionParserTest.java

```diff
@@ -1,12 +1,219 @@
 package com.google.step.youtube;
 
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Pattern;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-// @RunWith(JUnit4.class)
+@RunWith(JUnit4.class)
 public final class DescriptionParserTest {
 
+    /*                                                                   *
+     * ==================== GENERAL TESTS FOR PARSE ==================== *
+     *                                                                   */
+
+    /** String will return no matches from all regexs in parse() */
+    @Test
+    public void parse_MatchNone() {
+        String desc = "Check out our UPDATED version which has all the NEW ELEMENTS here:  https://youtu.be/rz4Dd1I_fX0"
+                        + "\nThe TEETH Song (Memorize Every Tooth): https://youtu.be/PI3hne8C8rU"
+                        + "\nDownload on ITUNES: http://bit.ly/12AeW99 ";
+
+        List<String> actual = DescriptionParser.parse(desc);
+        List<String> expected = Collections.emptyList();
+        Assert.assertEquals(expected, actual);
```

Comment:
Maybe inline the function calls instead of using 'expected' and 'actual'.  These seem self-explanatory/short  enough.

Reply:
ditto with the rest.

Reply:
Changed the parse tests to inline but the codeNoQuotes tests go over the 100 column limit so I moved just the expected values inline.